tests/test_configuration_parser.py

```diff
@@ -387,11 +394,31 @@ def test_check_flag_column_convert_to_list(self):
         flag_dict = config.check_flag_column()
         eq_(flag_dict, {"advisories": ['0']})
 
+
+    def test_check_flag_column_convert_to_list_test(self):
+        config = Configuration({"flag_column": {"advisories": "0"}})
+        flag_dict = config.check_flag_column(partition='test')
+        eq_(flag_dict, {"advisories": ['0']})
```

Comment:
I wonder if these tests should check the actual text of the warning, to make sure that we are getting the correct message for each `partition` type. If that makes sense to you, I can try to update the tests.